fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -815,6 +817,71 @@ impl From<MachNumber> for f64 {
     }
 }
 
+impl MaxDifference for MachNumber {
+    fn max_diff(self, other: Self) -> f64 {
+        (f64::from(self) - f64::from(other)).abs()
+    }
+}
+
+impl MachNumber {
+    // All formulas from Jet Transport Performance Methods by Boeing (March 2009 revision)
+
+    /// Get the pressure ratio for a given pressure
```

Comment:
Maybe be more specific that this function returns the pressure ratio to the standard pressure.